src/bootstrap.php

```diff
@@ -226,6 +226,7 @@ public function init() {
 		$this->check_system_status();
 		$this->export();
 		$this->webhooks();
+		$this->zapier( $this->log );
```

Comment:
You don't need to pass $log around here. It's accessible throughout the entire bootstrap.php file